plugins/oderave/odephysics.h

```diff
@@ -627,17 +671,34 @@ The possible properties that can be set are: ";
             }
         }
 
+        // get frictions before processing collisions.
+        float combinedFriction;
+        if (_usePerSurfaceFrictions) {
+            // we assume that a friction is a property of a link and hence every geom in a link has the same friction value.
```

Comment:
```suggestion
            // we assume that friction is a property of a link and hence every geom in a link has the same friction value.
```
nit. But also, if we assume the link has uniform friction, why not store it with the link?